### DISCUSSION TITLE
🇩🇪 Maniac Mansion

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/eltikia.github.io#4] [Q&A] [answered]
Was bedeute die Videobeschreibungen?

**Comment by Stagyrite** [comment, marked as answer]
Diese Beschreibungen macht Spaß.

### [Maniac Mansion](https://www.youtube.com/watch?v=UlD1jRwcvhA "🇩🇪 Maniac Mansion - YouTube")

> Lese eine Plakette. „Si trouve, envoyez-le au Louvre, poste paye.“ Das bedeute, wenn Sie es finden, senden Sie es portofrei an den Louvre.

Ich lernte ein bisschen Französisch, aber jetzt, ich habe keine Zeit, Französisch zulernen. Ich hatte mit diesen französisch Satz einen Nostalgie-Moment. Diese Beschreibungen können die Spielen wirklich nicht erklären.
